receiver/googlecloudspannerreceiver/README.md

```diff
@@ -53,13 +53,13 @@ receivers:
 
 Brief description of configuration properties:
 - **googlecloudspanner** - name of the Cloud Spanner Receiver related section in OpenTelemetry collector configuration file
-- **collection_interval** - this receiver runs periodically. Each time it runs, it queries Google Cloud Spanner, creates metrics, and sends them to the next consumer (default: 1 minute). **It is not recommended to change the default value of collection interval, since new values for metrics in the Spanner database appear only once a minute.**
+- **collection_interval** - this receiver runs periodically. Each time it runs, it queries Google Cloud Spanner, creates metrics, and sends them to the next consumer (default: 1 minute). **It is not recommended change the default value of collection interval, since new values for metrics in the Spanner database appear only once a minute.**
 - **top_metrics_query_max_rows** - max number of rows to fetch from Top N built-in table(100 by default)
 - **backfill_enabled** - turn on/off 1-hour data backfill(by default it is turned off)
 - **cardinality_total_limit** - limit of active series per 24 hours period. If specified, turns on cardinality filtering and handling. If zero or not specified, cardinality is not handled. You can read [this document](cardinality.md) for more information about cardinality handling and filtering.
 - **projects** - list of GCP projects
     - **project_id** - identifier of GCP project
-    - **service_account_key** - path to service account JSON key
+    - **service_account_key** - path to service account JSON key. It is highly recommended set this property to correct value. In case it is empty [Application Default Credentials](https://google.aip.dev/auth/4110) will be used for database connection.
```

Comment:
```suggestion
    - **service_account_key** - path to service account JSON key. It is highly recommended to set this property to the correct value. In case it is empty, the [Application Default Credentials](https://google.aip.dev/auth/4110) will be used for the database connection.
```

Reply:
Done.